redisinsight/ui/src/components/base/display/link/RiLink.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { LinkProps } from '@redis-ui/components'
+import { StyledLink } from './link.styles'
+
+export const RiLink = ({ color, ...props }: LinkProps) => (
```

Comment:
Follow up on the following suggestion https://github.com/redis/RedisInsight/pull/4841/files#r2292865447

Looking at this example, here we don't alter the base interface in any way, but since we're implementing RiComponents as a middle layer between RedisInsight and the outer world, does it make sense to define our own interfaces for each component we wrap? 

```
export interface RiLinkProps extends LinkProps {}

export const RiLink = ({ color, ...props }: RiLinkProps) => (
```

This way, our codebase will be bound to `RiLinkProps` and in case we need to switch the components library (once again), we'll do it in a single place (in theory).

Yes, it will look a bit meaningless, but I'm thinking out loud.